ch08.adoc

```diff
@@ -70,58 +70,76 @@ This information implies that the salinity field should be uncompressed to an ar
+The relationship between a tie point interpolation dimension and its corresponding interpolation dimension is defined with a __tie point index variable__. This contains zero-based indices that relate each element of a tie point interpolation dimension to its related location in the corresponding interpolation dimension. The tie point index variable is a one-dimensional integer variable that must span the tie point interpolation dimension specified by the  **`tie_point_dimensions`** attribute. The values must be strictly monotonically increasing within interpolation areas, and two adjacent indices that are equal, or differ by one, indicates the location of an interpolation area boundary relating to an grid discontinuity (<<compression-by-coordinate-tie-points>>).
 
-The tie point indices variable contains, for each tie point, the zero-based indices of the related point in the target domain. The tie point indices is an integer variable of the corresponding tie point dimension. Continuing the above example, the tie point indices variable **`int x_indices(tp_xc)`** could contain the following indices **`x_indices = 0, 9, 19, 29`** of the target domain. 
+When tie point variables represent a subset of the uncompressed coordinates, each value of the tie point index variable is the index of the interpolation dimension that corresponds to the corresponding tie point interpolation dimension.
 
-For the interpolation dimensions, the relationships between target domain dimensions and the tie point dimensions are defined trough the **`tie_point_indices`** attribute in combination with the tie point indices variables. Each target domain dimension of the **`tie_point_indices`** attribute is related with the dimension of the corresponding tie point indices variable. 
+Conversely, when tie point variables represent a superset of the uncompressed coordinates, each value of the tie point index variable is the index of the tie point interpolation dimension that corresponds to the corresponding interpolation dimension. This situation could occur when a hierarchy of different resolution representations of data are stored in different data variables. Space can be saved by storing the highest resolution coordinates, and using tie point indices with an interpolation variable to derive the lower resolution coordinates. Such a superset is identifiable by there being a unique interpolation area and the size of the tie point interpolation dimension being strictly greater than than the size of interpolation dimension.
```

Comment:
@TomLav would you be able to weigh in on this point? We've introduced this way of describing how to supersample coordinates and want to make sure that this is in line with a real use case, and you are the person with the most knowledge in this area (at least in this group).